internal/sbi/producer/access_token.go

```diff
@@ -152,8 +155,13 @@ func AccessTokenScopeCheck(req models.AccessTokenReq) *models.AccessTokenErr {
 	}
 	if _, err = nfCert.Verify(opts); err != nil {
 		logger.AccTokenLog.Errorln("Certificate verify error: " + err.Error())
-		return &models.AccessTokenErr{
-			Error: "invalid_client",
+		// DEBUG
+		// In testing environment, this would leads to follwing error:
+		// certificate verify error: x509: certificate signed by unknown authority
+		if !strings.Contains(err.Error(), "unknown authority") {
```

Comment:
```
if strings.Contains(err.Error(), "unknown authority") {
    logger.AccTokenLog.Warnf("Certificate verify: %v", err)
} else {
    return &models.AccessTokenErr{
        Error: "invalid_client",
    }
}
```

Reply:
> ```
> if strings.Contains(err.Error(), "unknown authority") {
>     logger.AccTokenLog.Warnf("Certificate verify: %v", err)
> } else {
>     return &models.AccessTokenErr{
>         Error: "invalid_client",
>     }
> }
> ```

Would you prefer this adjustment, or do you feel the need for any modifications?

Reply:
```
if strings.Contains(err.Error(), "unknown authority") {
    logger.AccTokenLog.Warnf("Certificate verify: %v", err)
} else {
    logger.AccTokenLog.Errorf("Certificate verify: %v", err)
    return &models.AccessTokenErr{ Error: "invalid_client", }
}
```